lib/exercism/team_stream_filters.rb

```diff
@@ -72,7 +72,7 @@ def sql
     # rubocop:disable Metrics/MethodLength
     def views_sql
       <<-SQL
-      SELECT 1 AS id, COUNT(views.id) AS total
+      SELECT 'team_stream' AS id, COUNT(1) AS total
```

Comment:
OK, so you're saying that we're only going to have one row, and therefore the id can be whatever we want it to be. We don't use the id to display anything, and we're not grouping on anything.